### PR TITLE
kitty: update to 0.13.1.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by maxice8** [comment]
That patch seems necessary.

**Comment by ndowens** [comment]
0.13.2 has been released now

**Comment by ndowens** [comment]
Without patch, it builds. and the file in do_install() no longer exists. I created my own template:
https://termbin.com/8yczk Used the find cmd due to where the location of kitty.conf is now in  a long numeric folder(reminds me of a sha checksum)

**Comment by Cogitri** [comment]
> Used the find cmd due to where the location of kitty.conf is now in a long numeric folder(reminds me of a sha checksum)

I use find too.

> Without the patch, it builds

Not on Travis or anything that uses many build jobs though

**Comment by Cogitri** [comment]
> 0.13.2 has been released now

Thanks for the notification though.

File: srcpkgs/kitty/0001-kitty-update-to-0.13.1.patch [hunk 1]
```diff
@@ -0,0 +1,47 @@
+From 678236ab64a3b2f07fa719b705bb7860b927218d Mon Sep 17 00:00:00 2001
```
Comment (maxice8) [inline srcpkgs/kitty/0001-kitty-update-to-0.13.1.patch]: uhhh?

**Pull request merged**